consensus/hotstuff/committee.go

```diff
@@ -38,12 +38,20 @@ type Replicas interface {
 	//   * model.ErrViewForUnknownEpoch if no epoch containing the given view is known
 	LeaderForView(view uint64) (flow.Identifier, error)
 
-	// WeightThresholdForView returns the minimum weight required to form a QC
+	// QuorumThresholdForView returns the minimum total weight of votes required to form a QC
 	// for the given view. This weight threshold is computed using the total weight
 	// of the initial committee and is static over the course of an epoch.
 	// Returns the following expected errors for invalid inputs:
 	//   * model.ErrViewForUnknownEpoch if no epoch containing the given view is known
-	WeightThresholdForView(view uint64) (uint64, error)
+	QuorumThresholdForView(view uint64) (uint64, error)
+
+	// TimeoutThresholdForView returns the minimum total weight of observed timeout objects
+	// required to safely timeout for the given view. This weight threshold is computed
+	// using the total weight of the initial committee and is static over the course of
+	// an epoch.
+	// Returns the following expected errors for invalid inputs:
+	//   * model.ErrViewForUnknownEpoch if no epoch containing the given view is known
+	TimeoutThresholdForView(view uint64) (uint64, error)
```

Comment:
:warning: To create TC we need the same threshold as for QC(2f+1). We need f+1 threshold for a specific liveness optimization called Bracha timeout mechanism. In our architecture proposal, Algorithm 2(Pacemaker) emits local timeout whenever it observes f+1 timeouts, this is still not enough to construct TC(2f+1) but based on quorum intersection lemma, already enough to understand that quorum eventually will be reached. Lemma 8 uses this property. 

Hopefully this context is useful. We need to change naming and comment.